lib/fog/openstack/models/identity_v3/projects.rb

```diff
@@ -8,8 +8,20 @@ class V3
         class Projects < Fog::OpenStack::Collection
           model Fog::Identity::OpenStack::V3::Project
 
+          @@cache = {}
+          Fog::Identity::OpenStack::V3::Project.use_cache(@@cache)
+
           def all(options = {})
-            load_response(service.list_projects(options), 'projects')
+            cached_project, expires = @@cache[{token: service.auth_token, options: options}]
```

Comment:
Could you send me some test code that exhibits the breaking behaviour? That will make it easier for me to reproduce & fix.